seqr/views/utils/variant_utils.py

```diff
@@ -329,56 +329,36 @@ def get_saved_discovery_variants_by_family(variant_filter, format_variants, get_
 
 def get_variant_inheritance_models(variant_json, family_individual_data):
```

Comment:
Can you add a comment with a summary of what the function is doing, and maybe type hints for args and return value?